src/monitor.js

```diff
@@ -17,7 +17,7 @@ class Monitor {
     this._resourceInterval = null;
   }
 
-  async reportError(err, level='error') {
+  async reportError(err, level='error', tags={}) {
```

Comment:
@imbstack I might very want to set tags every time...

Like:
`monitor.reportError(err, 'warning', {location: 'deadline-reaper-poll-loop'});`
As a way for me to attach a tiny bit of additional information that can tell where this error came from... or:
`monitor.reportError(err, {note: 'this happened somewhere in api.js where, not causing issues but something is very wrong as an internal invariant is broken... search for this message in the code...'});`